scripts/svn_shield_helper.rb

```diff
@@ -159,3 +159,34 @@ def getRanking(repo, release=false)
   rankHash
 
 end
+
+
+def dependencyBadge(repo, destdir, release=false)
+  site_config = YAML.load_file("./config.yaml")
+  numeric_version = (release) ? site_config['release_version'] : site_config['devel_version']
+  json_file = (repo == 'experiment') ? File.join("assets", "packages", "json", numeric_version, "data", "experiment","packages.json") : File.join("assets", "packages", "json", numeric_version, repo, "packages.json")
+  json_obj = JSON.parse(File.read(json_file))
+  for pkg in json_obj.keys
+    puts "#{pkg}"
+    info = json_obj[pkg]
+    shield = File.join(destdir, "#{pkg}.svg")
+    if (info.key?("dependencyCount"))
+      cnt = info["dependencyCount"]
+      puts "#{pkg} : #{cnt}"
+      resp = HTTParty.get("https://img.shields.io/badge/dependencies-#{cnt}-blue.svg")
```

Comment:
Does this make all badges blue? Maybe we could revisit this (later); it would be interesting to see a table of package, dep-count, look at the distribution..., maybe there's some number that starts to seem excessive and we could go to a green / yellow / red gradient

Reply:
yes it makes them all blue -  I figured a first pass would be to make it just an "informative" badge with no special coloring -  I can make some code next week to table the dependency counts to see some stats